conf/modules.config

```diff
@@ -84,6 +84,15 @@ process {
         ext.args = '--evalue 1.0e-25  --max-target-seqs 10 --max-hsps 1'
     }
 
+    withName: 'CREATE_BLOBDIR' {
+        ext.args = '--evalue 1.0e-25 --hit-count 10 --threads 4'
```

Comment:
Best to move the `--threads` to the actual module and make it dynamic. What happens if someone only has 2 cores?

Reply:
Moved, should we put this somewhere in the documentation too?